src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs

```diff
@@ -65,6 +65,11 @@ protected override void BeginProcessing()
         {
             mdOption = SessionState.PSVariable.GetValue("MarkdownOptionInfo", new MarkdownOptionInfo()) as MarkdownOptionInfo;
```

Comment:
I prefer `MarkdownOptionInfo` since the cmdlet is `Get-MarkdownOption`. @daxian-dbw  your thoughts?

Reply:
I think for any variables that are part of PowerShell itself, it should have the PS prefix to avoid name collision.  So this should be `PSMarkdownOptionInfo`.  cc @joeyaiello

Reply:
The variable `MarkdownOptionInfo` should be saved in the module scope of the Utility module, instead of the current scope of the current session state. Doing the latter will make the persistence of markdown option unreliable/inconsistent.

And, did you consider other options to persist the markdown option? I don't have an alternative off the top of my mind, but would like to check if you considered anything else.

Reply:
@SteveL-MSFT if we agree it should be moved to the module state, does it matter? 

As for persistence, I think it's okay to tell people to put this stuff in their profile until we hear why that's not good enough (seems to work just fine for PSReadline).

Reply:
@daxian-dbw  The committee feedback was to have it in session state so it is valid throughout the session. The preferences can be set in profile for persistence.

Reply:
I'm OK to have it in session state, but it should be the module session state of `Microsoft.PowerShell.Utility` instead of the currently active session state, which could be the global session state or another module's session state. And the scope to save the variable should be the module scope, not the current scope.

For example, if I'm running `Set-MarkdownOptions` in a function from another module, then the variable will be saved in the function scope under the session state of that module, which will be lost once the function finishes execution.